### DISCUSSION TITLE
Accessing Env Variables

### DISCUSSION THREAD

**Discussion by favorodera** [nuxt/nuxt#28419] [Questions] [answered]
![image](https://github.com/user-attachments/assets/34cbab07-1d02-4d38-a017-b22070e3c9b7)

Please i am having issues accessing my env variables from here.
The file is my firebase app config located at the root of my directory.
i have tried enabling asyncContext,using import.meta.env and process.env yet my app cant grab the variables unless i expose them

**Comment by oneminch** [comment, marked as answer]
Since `useRuntimeConfig` is a composable, I don't think you can call it outside some contexts. It can be called in `.vue` files, plugins, server routes, other composables, and middleware. There could be other contexts in which it is allowed, but it's not allowed in a root-level config file. I think creating a plugin to set up your Firebase config would be a better approach.

**Reply by favorodera** [reply]
Ok thanks
